egs2/aishell3/tts1/conf/finetune.yaml

```diff
@@ -0,0 +1,78 @@
+# This configuration is for ESPnet2 to finetune Tacotron 2. Compared to the
```

Comment:
Please remove this one and create `conf/train.yaml` which is symlink to `tuning/train_gst+xvector_tacotron2.yaml`.